src/lib/libfs.js

```diff
@@ -745,10 +745,11 @@ FS.staticInit();
     // Creates a whole directory tree chain if it doesn't yet exist
     mkdirTree(path, mode) {
       var dirs = path.split('/');
-      var d = '';
+      var d = path.startsWith('/') ? '/' : '';
       for (var dir of dirs) {
         if (!dir) continue;
-        d += '/' + dir;
+        if (d) d += '/';
+        d += dir;
```

Comment:
If `d` starts with `/`, say `/foo/bar`, then before the loop we end up with an initial value of `d = '/'`. Then in this first iteration we add `/' (since `d` is truthy) and end up with `//', after which we add `dir`, so we end up with `//foo` in the first iteration, which does not seem right?